### PR TITLE
docs: dropdown add show code in each example

### LINKED ISSUE #1156: Dropdown component should have Show Code section in each example
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
Dropdown component
-missing `Show Code` section in each example.
-in the Default Dropdown section, first dropdown should be `Cozy Dropdown` instead of `Compact Dropdown`. 
see screenshot:
Screen Shot 2019-08-20 at 10 42 45 AM

### PR DESCRIPTION
**Pull request by JYSW380**
#### Please provide a link to the associated issue.
fix #1156

#### Please provide a brief summary of this pull request.
-added Show Code section in each example.
-changed Cozy Dropdown in the Default Dropdown section

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
         "@types/node": "^12.7.2",
         "core-js": "^2.6.9",
         "focus-trap": "^5.0.2",
+        "fundamental-ngx": "^0.9.0",
```
Comment (JKMarkowski) [inline package.json]: Please remove it and rebuild package-lock.json.

File: docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <fd-popover style="margin-right: 12px;">
-    <fd-dropdown-control>Compact Dropdown</fd-dropdown-control>
+    <fd-dropdown-control>Cozy Dropdown</fd-dropdown-control>
```
Comment (JKMarkowski) [inline docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html]: Keep `Compact`.

File: package.json (hunk 1 above)
Comment (droshev) [reply package.json]: I would say revert the change

File: docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html (hunk 2 above)
Comment (InnaAtanasova) [reply docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html]: Should be `Cozy`, shouldn't it? Both examples say `Compact` when one should be `Compact` and the other should be `Cozy`

Comment (JYSW380) [reply docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html]: Please see screenshot for difference size of dropdown
Screen Shot 2019-08-22 at 10 20 18 AM

File: package-lock.json [hunk 3]
```diff
@@ -4709,6 +4709,17 @@
...
+            "version": "0.9.0",
+            "resolved": "https://registry.npmjs.org/fundamental-ngx/-/fundamental-ngx-0.9.0.tgz",
+            "integrity": "sha512-ZWEQV6tkFooERc+l0n1XNk62UyZJQutiaLOIIgDij1XPF31p3g0t3o//2i9FukSz6vGZ5BcH5+QtED6ITxVDdg==",
+            "requires": {
+                "focus-trap": "^5.0.2",
+                "fundamental-styles": "^0.1.0",
+                "popper.js": "^1.15.0",
+                "tslib": "^1.9.0"
+            }
+        },
```
Comment (InnaAtanasova) [inline package-lock.json]: This shouldn't be here. Can you remove it?

File: docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html (hunk 2 above)
Comment (JKMarkowski) [reply docs/app/documentation/component-docs/dropdown/examples/dropdown-default-example.component.html]: Ok you are completely right.

File: package-lock.json [hunk 4]
```diff
@@ -15808,4 +15808,4 @@
             "integrity": "sha512-mla2acNCMkWXBD+c+yeUrBUrzOxYMNFdQ6FGfigGGtEVBPJx07BQeJekjt9DmH1FtZek4E9rE1eRR9qQpxACOQ=="
         }
     }
-}
+}
```
Comment (JKMarkowski) [inline package-lock.json]: Please revert also this file.

**Comment by JYSW380** [comment]
@droshev @JKMarkowski 
reverted the files, pls approve again thanks :)

**Pull request merged**